### ISSUE TITLE
scoop update failed on wrong CApath

### ISSUE THREAD

**Issue by dlintw** [lukesampson/scoop#4211]
scoop update

```
Updating Scoop...
fatal: unable to access 'https://github.com/lukesampson/scoop/': error setting certificate verify locations:  CAfile: C:/Users/XXXX/scoop/apps/git/2.29.2.windows.3/mingw64/mingw64/ssl/certs/ca-bundle.crt CApath: none
Scoop update failed.
```

I found the issue is caused it should only one mingw64, there is two mingw64.  But, where to correct this issue?

**Comment by dlintw** [comment]
Solved by this way
`git config --global http.sslcainfo C:/Users/XXXX/scoop/apps/git/2.29.2.windows.3/mingw64/ssl/certs/ca-bundle.crt`

**Comment by david-bakin** [comment]
I had this problem too.  I had installed scoop/shovel on a box where Git For Windows was already installed.  scoop/shovel was pointing there for `ca-bundle.crt`.  I was using the scoop/shovel git throughout - it was first on the PATH.  So I uninstalled Git For Windows and proceeded to get this error.

IMO scoop/shovel should, after installing it's own git, set `http.sslcainfo` to something like `<scoop-root>/apps/git/current/usr/ssl/certs/ca-bundle.crt`, that is, point into it's own managed git.
